Packs/knowbe4Phisher/Integrations/knowbe4Phisher/knowbe4Phisher_description.md

```diff
@@ -4,4 +4,4 @@
 In order to generate a new API Key in your KnowBE4 console navigate to **Account Settings** > **Account Integrations** > **API** > **PhishER API** 
 
 ### Configuration params
-**NOTE**: If using 6.0.2 or lower version, put your API Key in the **Password** field, leave the **apikey** field empty.
+**NOTE**: If using 6.0.2 or lower version, put your API Key in the **Password** field, leave the **user** field empty.
```

Comment:
This is a mistake no? the field is called 'apikey'

Reply:
yes